aiida_quantumespresso/calculations/pwimmigrant.py

```diff
@@ -242,7 +268,14 @@ class or using the ``set_`` methods (e.g. ``set_remote_workdir``).
                                            self._OUTPUT_SUBFOLDER,
                                            '{}.save/'.format(self._PREFIX),
                                            fnm)
-                open_transport.get(remote_path, folder.abspath)
+                if open_transport.path_exists(remote_path):
+                    open_transport.get(remote_path, folder.abspath)
+                else:
```

Comment:
What if the pseudo's also cannot be found here? Would it perhaps be better to define a list of potential paths and loop over them and as soon as one is found to exist call `open_transport.get` and if none of the potential paths exist, we either raise or log an error. Given that QE allows to define a pseudo dir variable in the input to allow a single directory with pseudos, I imagine a lot of calculation folders will not have copies of the pseudo's. I am now thinking whether it makes sense to import a `PwCalculation` without the pseudo's as it won't be reproducible. So in the fall through case we should maybe always raise a specific exception, notifying that the pseudos could not be found. The user could then copy to the folder manually, or we can potentially provide a flag to ignore the error.

P.S. for the for loop you could use the [for-else construct in python](http://book.pythontips.com/en/latest/for_-_else.html#else-clause) and raise in the else clause: see

Reply:
yep makes sense